shared/settings/index.js

```diff
@@ -1,20 +1,43 @@
 // @flow
+import * as React from 'react'
 import SettingsContainer from './render'
 import pausableConnect from '../util/pausable-connect'
 import {switchTo} from '../actions/route-tree'
 import {logout} from '../actions/login/creators'
 
+import type {Tab} from '../constants/settings'
+import type {TypedState} from '../constants/reducer'
 import type {RouteProps} from '../route-tree/render-route'
 
-// $FlowIssue type this connector
-export default pausableConnect(
-  (state, {routeSelected, routeLeafTags}: RouteProps<{}, {}>) => ({
-    badgeNumbers: {}, // TODO add badging logic
-    selectedTab: routeSelected,
-    isModal: routeLeafTags.modal,
```

Comment:
isModal seems unused

Reply:
??? isn't this using it https://github.com/keybase/client/blob/264a242ee310ba7d5e6faa1a1722225176f9557d/shared/settings/render.desktop.js#L13

Reply:
Yes, don't know how I missed that. :(